chart/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -0,0 +1,53 @@
+replicaCount: 1
+
+image:
+  repository: eu.gcr.io/kyma-project/incubator/kyma-environment-broker
```

Comment:
Why don't you just put those values regarding the image in the main values.yaml file?